lib/biokbase/catalog/registrar.py

```diff
@@ -546,7 +536,7 @@ def build_docker_image(self, docker_client, image_name, basedir):
 
         # examine stream to determine success/failure of build
         imageId = None
-        for lines in docker_client.build(path=basedir, rm=True, tag=image_name, pull=False):
+        for lines in docker_client.build(path=basedir, rm=True, tag=image_name, pull=True):
```

Comment:
Will this cause problems for the old sdkbase image that shouldn't get pulled?

Reply:
Also is there any way to test this?

Reply:
The only time that might cause trouble is if the sdkbase image on docker is newer than the sdkbase image in the environment running the catalog. (I'd have to ask Keith to get that listing for each environment). The new images that we have been building are different namespace so should be OK.

I should be able to test this by looking that the docker image is not cached in the build log. To do that I need tests on the master branch to work however.

Reply:
> The only time that might cause trouble is if the sdkbase image on docker is newer than the sdkbase image in the environment running the catalog. (I'd have to ask Keith to get that listing for each environment)

That's what I'm concerned about - if we're not careful we could bork the sdkbase image.

Reply:
The sdkbase on docker hub is over a year old.  https://hub.docker.com/r/kbase/sdkbase/tags/

Keith sez:
>on prod, the kbase/kbase:sdkbase.latest is 10 months old, so it's probably v1
>
>(there is a kbase/sdkbase2:latest image there too)
>on ci, there's kbase/kbase:sdkbase2.latest and kbase/kbase:sdkbase.latest.  the latter is 6 >weeks old

The versions on the server are fresher than docker hub so I think we should be OK

Reply:
@sychan you're sdkbase docker image expert at this point, what say you?

Reply:
> I should be able to test this by looking that the docker image is not cached in the build log. 

So is it possible to add these tests now?